app/models/jobseeker_profile.rb

```diff
@@ -51,7 +51,7 @@ class JobseekerProfile < ApplicationRecord
 
   def self.copy_attributes(record, previous_application)
     record.assign_attributes(
-      employments: previous_application.employments.map(&:duplicate),
+      employments: previous_application.employments.map(&:clone),
```

Comment:
out of curiosity, why use `clone` here? Also the qualifications uses `duplicate`

Reply:
that should now be reverted - clone was a mistake. Turns out #dup is the correct way, but dup copies parent associations